packages/amplify_core/test/amplify_hub_test.dart

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'dart:async';
+
+import 'package:amplify_core/amplify_hub.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:amplify_core/amplify_core.dart';
+
+import 'resources/fake_stream_controller.dart';
```

Comment:
`resources` makes me think of static content, but this is a library utility. Can you just bundle it under `test/fake_stream_controller.dart`? If you need to use it across multiple modules, your test utils module might be a candidate home, as well.

Reply:
Fixed.